### PR TITLE
Add a news post for playtest-20150830

### PR DESCRIPTION
**Pull request by penev92**
Short and simple, but I think it's enough for this playtest.
Had a hard time coming up with a screenshot looking at the tiny changelog. I know it's a bit repetitive, but if anyone has a better idea they can take another one.

### CODE REVIEW

File: content/news/playtest-20150830.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Another playtest on the road to our next stable release is ready!
+
+Being in a feature freeze, we focused on fixing bugs and inconsistencies reported by community members to ensure the release is as bug-free as possible.
+
+More notable things that got fixed include:
+
+* An exploit with spies in RA introduced with the new spy ability.
```
Comment (pchote) [inline content/news/playtest-20150830.md]: Which new spy ability?

File: content/news/playtest-20150830.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Issues with crushing infantry and walls.
+* Issues with entering transports.
```
Comment (pchote) [inline content/news/playtest-20150830.md]: Maybe add "units" before entering.

File: content/news/playtest-20150830.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+For details check out the [full changelog](http://changelog.openra.net).
+
+Grab the installer for this playtest [from our download page](/download/) and give it a try!
+
+<div style="text-align:center" markdown="1">
+
+![Custom map rule warning](/images/news/20150830-new-ra-production-overlays.png)
+
+Production icons in RA got a new overlay that is more fitting of the RA95 style.
```
Comment (pchote) [inline content/news/playtest-20150830.md]: I'd word this as something like "The spy production veterancy overlay has been improved to better fit the RA icon style"

**Comment by pchote** [comment]
A couple of wording nits, but the content looks fine to me.

**Comment by Mailaender** [comment]
:+1:

**Comment by Mailaender** [comment]
Tagged https://github.com/OpenRA/OpenRA/releases/tag/playtest-20150830 and updated https://build.opensuse.org/package/show/games:openra/playtest so you only need to fill in the IDs here.

**Pull request merged**